draft-ietf-tls-tls13.md

```diff
@@ -1406,6 +1416,10 @@ ClientHello (without modification) except:
 
 - Including a "cookie" extension if one was provided in the
   HelloRetryRequest.
+
+- Replacing the "pre_shared_key" and "psk_binder" extensions with
+  the one indicated in the HelloRetryRequest if indicated by
+  a "pre_shared_key" entry in the HelloRetryRequest.
```

Comment:
I assume that this is provisional and you would prefer to go back to multiple PSKs.